juriscraper/opinions/united_states/federal_special/__init__.py

```diff
@@ -13,4 +13,5 @@
     "uscfc_u",
     "uscfc_vaccine",
     "uscfc_vaccine_u",
+    "uscg_ccap",
```

Comment:
@mlissner I think we should rename this to `uscg_cca`

Reply:
All of your notes look good to me, @flooie. I don't know if we've documented this, but if we put underscores in the file name, everything after the first underscore is ignored by CourtListener. So if we used `uscg_cca`, the court on CL would be `uscg`, which I don't think is what we want. I think on CL we'd want the court ID to be `uscgcca`, so the filename should be `uscgcca.py`.